tests/test_google_sheets_datatypes.py

```diff
@@ -155,12 +154,12 @@ def test_run(self):
         # Verify that all data has been coerced to the expected column datatype
         record_data = [message['data'] for message in synced_records[test_sheet]['messages'] if message.get('data')]
         data_map = {
-	    "Currency": Decimal, # BUG Currency is being identified as a decimal type rather than string https://jira.talendforge.org/browse/TDL-14360
+	    "Currency": str, # BUG Currency is being identified as a decimal type rather than string https://jira.talendforge.org/browse/TDL-14360
```

Comment:
Remove this bug comment because Currency is identifying as str as per code change.

Reply:
removed the comment